vmms/distDocker.py

```diff
@@ -8,6 +8,8 @@
 # `domain_name` attribtue of TangoMachine.
```

Comment:
As you have changed the default docker image to xenial the issue here would happen https://github.com/autolab/Tango/issues/138, 

do update with the patch from here
https://github.com/mtoupsUNO/Tango/commit/b97bc1e1eaa000644d4e030e44e1c96341c2d120

Reply:
Patch applied